### ISSUE TITLE
Font size in top bar (clock, application title) and shell prompts too small

### ISSUE THREAD

**Issue by RedBearAK** [vinceliuice/WhiteSur-gtk-theme#317]
Please provide your system info (distro version, etc.) and of course, the screenshot

Fedora 34, and Ubuntu 21.04

I haven't noticed this problem on a MacBook with a much lower resolution display (1280x800) but on a laptop with a 1080p FHD screen the font size in the top bar for things like the clock and the application title look quite small and thin. I see a similar font size issue with the buttons on GNOME shell dialogs such as the dialog that overlays the screen when you use "Restart..." or "Power Off..." from the GNOME status menu. 

Tried looking through the `gnome-shell.css` file but can't figure out what might apply to these areas. 

Here is a screenshot of the top bar with the "default" shell theme, followed by the top bar with WhiteSur-dark-solid set as the theme, and then a screenshot of the "Restart..." prompt with the default theme and then with WhiteSur-dark-solid set as the theme. You should see that the fonts in the top bar and in the prompt buttons seem disproportionately small, while the text inside the prompt is fine. 

![top_bar_default_shell_theme](https://user-images.githubusercontent.com/64876997/122838392-7ab2b880-d2a2-11eb-9aeb-7b16e9fe97ef.png)

![top_bar_whitesur-dark-solid_shell_theme](https://user-images.githubusercontent.com/64876997/122838444-93bb6980-d2a2-11eb-9a9e-7b848046efad.png)

![prompt_default_shell_theme](https://user-images.githubusercontent.com/64876997/122838028-a71a0500-d2a1-11eb-8f20-04ba266ac98d.png)

![prompt_whitesur-dark-solid_shell_theme](https://user-images.githubusercontent.com/64876997/122838060-bac56b80-d2a1-11eb-9ee5-66180f68d044.png)

**Comment by RedBearAK** [comment]
Also affects all the menus that drop down when you click on any items in the top bar. 

The only way I've found to change it is by using the "Large Fonts" option in Accessibility settings, but that makes everything a little bit too big and affects the entire UI.

**Comment by vinceliuice** [comment]
./install.sh -HD

**Comment by RedBearAK** [comment]
> ./install.sh -HD

I usually just install using ocs-url from gnome-look.org. 

I downloaded the latest source ZIP and tried to run that command and got this error: 

```
  ERROR: Unrecognized installation option '-HD'.

  Try './install.sh --help' for more information.
```

There is no "HD" or "hd" option that I can see in the help info. 

Are you perhaps referring to a feature existing in some other branch that hasn't been merged yet? 

Tried the `--interactive` and `--dialog` options and the theme was installed, but there was no interactivity or dialogs. The font seems thicker than before but still small.

**Comment by vinceliuice** [comment]
![Screenshot_20210622_085823](https://user-images.githubusercontent.com/7604295/122846271-15110b80-d338-11eb-8056-cdbed9110ad5.png)

**Comment by RedBearAK** [comment]
OK, this time I tried downloading using the big green "Code" button on the main GitHub page to just download a ZIP of the current master branch. Now suddenly it works. It didn't work on the latest release version from 5-05 that I downloaded from the Releases page. 

Maybe it's time for a new release. 

Font size and weights are much better now. The drop-down menus also look more appropriate. The prompts look good too. 

Thanks. This was really the only thing I've ever found "off" with this theme, and it's otherwise the best theme I've been able to find for GNOME desktops. Nice that it's available for KDE too, and integrates with Flatpaks and Snaps.
